packages/eslint-plugin-pf-codemods/lib/rules/v5/tabs-warn-children-type-changed.js

```diff
@@ -0,0 +1,23 @@
+const { getPackageImports } = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/8217
+module.exports = {
+  create: function (context) {
+    return {
+      ImportDeclaration(node) {
+        const TabsImport = node.specifiers.find(
+          (specifier) =>
+            specifier.imported.name === "Tabs" &&
+            node.source.value === "@patternfly/react-core"
+        );
+
+        if (TabsImport) {
+          context.report({
+            node,
+            message: "Children of the 'Tabs' component must now be 'Tab' components or expressions resulting in a falsy value.",
```

Comment:
I don't _love_ this message, but it was the best I could come up with. 100% open to suggestions.

Reply:
idk why but when i see "children of the" i think "corn" 😆 

how about "`Tabs` component children must be `Tab` components"

not sure that the falsy needs to be addressed. technically it would be "expressions resulting in either falsy value or a Tab component"

Reply:
"The `children` prop of the Tabs component must now be passed a Tab component or a falsy value."?

Reply:
@thatblindgeye  I like the sound of that, but since `children` isn't usually used explicitly as a prop I wonder if that might trip people up 🤔 

@gitdallas so your suggestion would be "Tabs component children must be Tab components or expressions resulting in either falsy value or a Tab component" in its full form?

Reply:
@wise-king-sullyman - i really don't think the mention of expressions is necessary, anything could be expression but that doesn't matter... all that matters is what comes out of the expression. 

i was thinking "`Tabs` component children must be `Tab` components or a falsy value", I guess... or in Eric's format we could have "The `children` of the `Tabs` component must now be passed a `Tab` component or a falsy value."